# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/asahi-audio/hotfix-install-t.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- asahi-audio-0.5/Makefile.vanilla	2023-11-26 14:14:04.157453904 +0100
+++ asahi-audio-0.5/Makefile	2023-11-26 14:14:43.766451604 +0100
@@ -41,7 +41,7 @@
 
 j%: core
 	install -dDm0755 $(DESTDIR)/$(DATA_DIR)/asahi-audio/$@/
-	install -pm0644 -t $(DESTDIR)/$(DATA_DIR)/asahi-audio/$@/ $(wildcard firs/$@/*)
+	install -pm0644 $(wildcard firs/$@/*) $(DESTDIR)/$(DATA_DIR)/asahi-audio/$@/
 
 
 install: aliases core j274 j293 j313 j314 j316 j415
